paasta_tools/paasta_metastatus.py

```diff
@@ -204,7 +218,7 @@ def assert_disk_health(metrics, threshold=10):
         )
 
 
-def assert_tasks_running(metrics):
+def assert_tasks_running((metrics, mesos_state)):
```

Comment:
no, these are because the ``run_healthchecks_with_param`` expects a single parameter, and these functions now need both the ``mesos_state`` and ``metrics`` params, so @nhandler is wrapping them in a tuple to keep obeying the 'single param' requirement.

This is actually just hinting that the abstraction here is no longer really useful. Long term, we'll get rid of it.

@nhandler my suggestion here would be to drop the usage of ``run_healthchecks_with_param`` from ``get_mesos_metrics_health`` completely. Instead, replace it's implementation with:

```
def get_mesos_metrics_health(mesos_metrics, mesos_state):
    """Perform healthchecks against mesos metrics.
    :param mesos_metrics: a dict exposing the mesos metrics described in
    https://mesos.apache.org/documentation/latest/monitoring/
    :returns: a list of HealthCheckResult tuples
    """
   return [func(mesos_metrics, mesos_state) for func in [
        assert_cpu_health,
        assert_memory_health,
        assert_disk_health,
        assert_tasks_running,
        assert_slave_health,
    ]
```